src/PiWeb.Import.Sdk/Modules/ImportFormat/ImportAction.cs

```diff
@@ -0,0 +1,56 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+namespace Zeiss.PiWeb.Import.Sdk.Modules.ImportFormat;
+
+/// <summary>
+/// Represents the result of <see cref="IImportFormat.DecideImportAction"/> and defines how the <see cref="IImportGroup"/> should be handled by
+/// PiWeb Auto Importer.
+/// </summary>
+public enum ImportAction
+{
+    /// <summary>
+    /// <see cref="IImportGroup.PrimaryFile"/> has an unknown format and <see cref="IImportGroup"/> should be offered to another
+    /// <see cref="IImportFormat"/>.
+    /// </summary>
+    Pass,
+
+    /// <summary>
+    /// The <see cref="IImportGroup"/> contains all relevant information and can be imported at once.
+    /// </summary>
+    Import,
+
+    /// <summary>
+    /// <see cref="IImportGroup.PrimaryFile"/> was recognized but the <see cref="IImportGroup"/> should be discarded, e.g. because a file
+    /// is corrupted. The files in <see cref="IImportGroup.AllFiles"/> will not be offered to another import format and will be removed
+    /// from the import folder.
+    /// </summary>
+    Discard,
+
+    /// <summary>
+    /// <see cref="IImportGroup.PrimaryFile"/> was recognized and the <see cref="IImportGroup"/> is ready to be imported but there is
+    /// information missing. This information might be available later.
+    /// The files in <see cref="IImportGroup.AllFiles"/> will not be offered to another import format in the ongoing import run, but they
+    /// will be available in the next import run again. 
+    /// If the retention period of the files in <see cref="IImportGroup.AllFiles"/> is expired the <see cref="IImportGroup"/> is imported
+    /// with the available information.
+    /// </summary>
+    RetryOrImport,
+
+    /// <summary>
+    /// <see cref="IImportGroup.PrimaryFile"/> was recognized and the <see cref="IImportGroup"/> is ready to be imported but there is
+    /// information missing. This information might be available later.
+    /// The files in <see cref="IImportGroup.AllFiles"/> will not be offered to another import format in the ongoing import run, but they
+    /// will be available in the next import run again.
+    /// If the retention period of the files in <see cref="IImportGroup.AllFiles"/> is expired the <see cref="IImportGroup"/> is discarded
+    /// and the files are removed from the import folder.
+    /// </summary>

```

Comment:
```
   /// <summary>
    /// <see cref="IImportGroup.PrimaryFile"/> was recognized and the <see cref="IImportGroup"/> contains not enough information to be imported at once,
    /// but can wait for additional information that might appear later.
    /// So the <see cref="IImportGroup"/> should only be discarded if the retention period of the files in <see cref="IImportGroup.AllFiles"/> is expired.
    /// If the retention period is not expired, the files in <see cref="IImportGroup.AllFiles"/> will not be discarded in the ongoing import run
    /// but will be available again in the next import run. 
    /// </summary>
```

Reply:
I changed the first part of the text al little bit and used your suggestion for the second part.